trackpy/feature.py

```diff
@@ -771,11 +795,11 @@ def batch(frames, diameter, minmass=100, maxsize=None, separation=None,
                      filter_before=filter_before, filter_after=filter_after)
 
     if meta:
-        if isinstance(meta, str):
-            filename = meta
+        if isinstance(meta, six.string_types):
+            with open(filepath_or_obj, 'w') as file_obj:
```

Comment:
shouldn’t this be `open(meta, 'w') ... ` ?  

I am also not a huge fan of blind writes (which will nuke what ever is there).  However, apparently the 'barf if file exists at that path' mode is a hdf5-ism, not part of the standard `open` so should probably ignore this complaint for now.